xla/service/gpu/fusions/input_slices_mlir_test.cc

```diff
@@ -41,8 +41,8 @@ TEST_F(MlirInputSlicesFusionTest, SimpleInputSlices) {
     }
   )";
   TF_ASSERT_OK(EmitAndCheckIR(kHloString, R"(
-    // CHECK: arith.cmpi sge
-    // CHECK: arith.cmpi sle
+    // CHECK: arith.cmpi s{{[gl]}}e
+    // CHECK: arith.cmpi s{{[gl]}}e
```

Comment:
I guess this is a matter of different order of sge and sle? Then it would be better to change to CHECK-DAG for those two lines.

Reply:
I'll try that

Reply:
Done.
Tangential question: is there some easy way to tell the compiler to limit to 512 threads for one of these tests? There is a llvm attribute amdgpu-flat-work-group-size but I can't figure out how to pass it to the test. In a few other tests, there are substantial differences between NV and AMD versions, because the compiler uses 512 for NV and 1024 for AMD.